### PR TITLE
reverting to original feedstock rendering

### PR DESCRIPTION
**Pull request by buijennifer**
- Build No. 9
- Restoring feedstock files to match templates from conda smithy repo
- Showing that when 'upload_on_branch' is not defined in conda-forge.yml, files follow original templates
- Example for PR

### CODE REVIEW

File: recipe/meta.yaml
Comment (mrakitin) [inline recipe/meta.yaml]: [Conda-forge docs](https://conda-forge.org/docs/maintainer/adding_pkgs.html?highlight=pip%20install#optional-bld-bat-and-or-build-sh) suggest using pip instead:
```suggestion
   script: "{{ PYTHON }} -m pip install . --no-deps -vv"
```

**Review by mrakitin — CHANGES_REQUESTED** [review]
The first successful build before my suggested changes generated this set of packages, even before this PR is merged:
image

Please apply the suggestion and we will see that no new packages are uploaded with the changes from the review.

**Comment by buijennifer** [comment]
The latest builds were not uploaded because the package was already uploaded to anaconda.org.

https://dev.azure.com/buijennifer/nsls2/_build/results?buildId=197&view=logs&jobId=4f922444-fdfe-5dcf-b824-02f86439ef14&taskId=937c195f-508d-5135-dc9f-d4b5730df0f7&lineStart=649&lineEnd=650&colStart=1&colEnd=1

**Review by mrakitin — APPROVED** [review]
Thanks for the update.

**Pull request merged**
